Userland/Libraries/LibJS/Runtime/Array.h

```diff
@@ -45,6 +44,9 @@ class Array : public Object {
 
     [[nodiscard]] bool length_is_writable() const { return m_length_writable; };
 
+protected:
+    explicit Array(Object& prototype);
```

Comment:
Why protected and not private for this one?

Reply:
ArrayPrototype inherits Array :acid2kiss: